engines/python/setup/djl_python/utils.py

```diff
@@ -13,24 +15,37 @@ class ParsedInput:
     errors: dict
     batch: list
     is_client_side_batch: list = field(default_factory=lambda: [])
+    adapters: list = None
+    found_adapters: bool = False
+
+
+@dataclass
+class InputFormatter:
```

Comment:
Is the plan to extend the functionality of this class to do formatting? If not, I think we should change the name to something like InputFormatConfigs or RequestFormatConfigs

Reply:
I am not clear of the use case to extend this yet to have multiple formats yet. I like InputFormatConfigs, will change to this.

Reply:
Done